src/auth/index.ts

```diff
@@ -25,18 +25,22 @@ type EgoAppCredential = {
 };
 
 export type EgoJwtManager = {
-  getLatestJwt: () => Promise<EgoAccessToken>;
+  getLatestJwt: (options?: { noCache?: boolean }) => Promise<EgoAccessToken>;
 };
 
 export const createEgoJwtManager = async (): Promise<EgoJwtManager> => {
   let cachedJwt = await getJwt();
-  const getLatestJwt = async () => {
-    const egoTokenUtil = await createEgoUtil();
-    const decodedToken = egoTokenUtil.decodeToken(cachedJwt.access_token);
-    cachedJwt = egoTokenUtil.isExpiredToken(decodedToken)
-      ? await getJwt()
-      : cachedJwt;
-    return cachedJwt;
+  let egoTokenUtil = await createEgoUtil();
```

Comment:
Every time we create the egoUtils it will fetch the Ego public key. Let's only do this if `getLatestJwt` specifies to not use the cache.